.gitlab/benchmarks.yml

```diff
@@ -66,17 +66,27 @@ check-big-regressions:
 
     KUBERNETES_SERVICE_ACCOUNT_OVERWRITE: dd-trace-js
 
-benchmark-v14:
+benchmark:
   extends: .benchmarks
+  parallel:
```

Comment:
So nine jobs instead of three? We might need to tweak the repo settings / branch protections for this part.

Reply:
Unfortunately yes, because the benchmarking platform is not horizontally scalable on its own, so otherwise the maximum number of cores is reached (24 cores on CPU2). Splitting in 3 results in ~15 cores used per job.

Reply:
I think it's fine.

Before, if one of the three failed, then it's like "well it looks like Node.js v16 is an issue". In this case, if one of the nine fails, well it still means "it looks like Node.js v16 is an issue". The added granularity shouldn't result in more difficult debugging.

Reply:
Yeah, this is one of the reasons I split on the whole benchmark instead of the variants. Splitting on the variant would require 6 machines instead of 9, but then it would be more difficult to work with because the output would be split on multiple machines. I think this strikes a good balance and will scale well without being problematic for debugging.